minesim.go

```diff
@@ -373,28 +373,22 @@ func main() {
 		minedblocks += m.mined
 		totalstale += m.mined - m.credit
 	}
-	fmt.Printf("seed-arg %d\n", g.seed)
-	fmt.Printf("block-interval-arg %d\n", g.blockinterval)
-	fmt.Printf("stopheight-arg %d\n", g.stopheight)
-	fmt.Printf("total-hashrate-arg %.2f\n",
-		g.totalhash)
-	fmt.Printf("mined-blocks %d\n",
-		minedblocks)
-	fmt.Printf("height %d %.2f%%\n", totalblocks,
-		float64(totalblocks)*100/float64(minedblocks))
-	fmt.Printf("total-simtime %.2f\n",
-		g.currenttime)
-	fmt.Printf("ave-block-time %.2f\n",
-		float64(g.currenttime)/float64(totalblocks))
-	fmt.Printf("total-stale %d %.2f%%\n",
-		totalstale, float64(totalstale*100)/float64(minedblocks))
-	fmt.Printf("max-reorg-depth %d\n", g.maxreorg)
+	fmt.Printf("%-20s %-d\n", "seed-arg", g.seed)
+	fmt.Printf("%-20s %-d\n", "block-interval-arg", g.blockinterval)
+	fmt.Printf("%-20s %-d\n", "stopheight-arg", g.stopheight)
+	fmt.Printf("%-20s %-.2f\n", "total-hashrate-arg", g.totalhash)
+	fmt.Printf("%-20s %-d\n", "mined-blocks", minedblocks)
+	fmt.Printf("%-20s %-d \t%-.2f%%\n", "height", totalblocks, float64(totalblocks)*100/float64(minedblocks))
+	fmt.Printf("%-20s %-.2f\n", "total-simtime", g.currenttime)
+	fmt.Printf("%-20s %-.2f\n", "ave-block-time", float64(g.currenttime)/float64(totalblocks))
+	fmt.Printf("%-20s %-d \t%-.2f%%\n", "total-stale", totalstale, float64(totalstale*100)/float64(minedblocks))
```

Comment:
Since nearly everything else is expressed as a percentage, and since the absolute number isn't very meaningful, let's just simplify this to be only a percentage. Then (combined with the above change) this part of the output is very clean, no tab characters, and just one value per line.
```suggestion
	fmt.Printf("%-20s %-.2f%%\n", "total-stale", float64(totalstale*100)/float64(minedblocks))
```